internal/pkg/siderolink/manager.go

```diff
@@ -219,6 +219,7 @@ func (p *Params) NewListener() (net.Listener, error) {
 
 	return tls.Listen("tcp", p.APIEndpoint, &tls.Config{
 		Certificates: []tls.Certificate{cert},
+		NextProtos:   []string{"h2"},
```

Comment:
this is not a machine API? not sure what kind of machine API you're looking into?

this is gRPC API so, it's always using HTTP/2